online-shop/src/components/product/product.component.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+// import { preProcessFile } from 'typescript';
+
+import "./product.styles.css"
+
+
+export interface Produs {
+    id:number,
+    name:String,
+    category:String,
+    price: number,
+    imageUrl: String
+}
+
+// m-am oprit aici cu style ul pana ma uit peste materialUI :(
+// nu e responsive inca
```

Comment:
Don't waste a lot of time with responsiveness, focus more on the actual functionalities of  both MaterialUI and JSS